src/mars.d

```diff
@@ -1942,4 +1942,25 @@ private void addDefaultVersionIdentifiers()
     if (global.params.useArrayBounds == BOUNDSCHECKoff)
         VersionCondition.addPredefinedGlobalIdent("D_NoBoundsChecks");
     VersionCondition.addPredefinedGlobalIdent("D_HardFloat");
+
+    printPredefinedVersions();
+}
+
+private void printPredefinedVersions()
+{
+    import core.stdc.stdio;
+    if (global.params.verbose && global.params.versionids) {
+        printf("predefs  ");
```

Comment:
Actually we don't want that, otherwise we would ruin the alignment with the rest of the output.